packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.9.18+14
+
+* Handle video and audio interruptions and errors.
+* Use a single time offset for both video and audio.
```

Comment:
... to fix a bug where audio and video are out of sync after interrupted by phone cals.